snuba/web/views.py

```diff
@@ -373,22 +375,19 @@ def sdk_distribution(*, timer: Timer) -> Response:
     return format_result(run_query(dataset, request, timer))
 
 
-@application.route("/subscriptions", methods=["POST"])
-def create_subscription():
+@application.route("/<dataset:dataset>/subscriptions", methods=["POST"])
+def create_subscription(*, dataset: Dataset):
     return (
         json.dumps({"subscription_id": uuid1().hex}),
```

Comment:
This return value will likely need to change, but need to coordinate with @wedamija on what makes sense here.

Reply:
I'll just handle this in my pr